sda/internal/database/database.go

```diff
@@ -46,6 +46,7 @@ type SyncData struct {
 }
 
 type SubmissionFileInfo struct {
+	FileID    string `json:"fileID"`
```

Comment:
This struct is also used when the submission user wants to list the files that have been uploaded and the `FileID` is not something that the submitter needs to see.

Reply:
But is it ok if they still do..? Otherwise we will need to make a separation so that helpdesk and admins can see it and other users cannot.

Reply:
After the discussion on Slack, I think we should leave this for now. Feel free to create an issue if you would like it to be changed 👍